Server/src/models/VariableModel.ts

```diff
@@ -0,0 +1,25 @@
+import { Datapoints } from "./entities/Datapoints";
+import { IVariableNameModel } from "../../../Client/src/Models/IVariableNameModel"
+import { getConnection, getManager } from "typeorm";
+
+
+export class VariableModel {
+  /**
+  * This method will return all existing variables or datapoints in the format of IVariableModel
+  */
+  async getAllVariables(): Promise<IVariableNameModel[]> {
+    let variables = await getManager()
+      .createQueryBuilder(Datapoints, 'Datapoints')
+      .select('name')
+      .distinct(true)
+      .getRawMany();
+
+    let variableModel = variables.map(variable => {
+      let variableModel: IVariableNameModel = {
```

Comment:
Can you rename this? Its pretty confusing using the same variable name

Reply:
Removed it